<?xml version="1.0"?>

        <!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
        <!--                                                                                    -->
        <!--                                                                                    -->
        <!--                            LAUNCH SENSOR PLACEMENTS MAP                            -->
        <!--                                                                                    -->
        <!--                                                                                    -->
        <!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
        <!--                              @AsifArain, Dec 08, 2017                              -->
        <!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->

<launch>


    <!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
    <!--                              environment map                                       -->
    <!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
	<node pkg="plan_execution" type="gdm_pcl_publisher_node" name="gdm_pcl_publisher" output="screen">
	
	    <param name="map_file"	        value="reconstruction.dat"/>
        <param name="map_size_file"	    value="reconstruction_mapsize.dat"/>
        <param name="cell_size_file"	value="reconstruction_cellsize.dat"/>
        <param name="robot_origin_file"	value="reconstruction_origin.dat"/>
        <param name="high_threshould"	value="500"/>
        <!--param name="conf_file"	        value="robot_plan_detection_test.dat"/-->
        
        <param name="hard_offset_x"	    value="0.0"/> <!--"0.0"-->
        <param name="hard_offset_y"	    value="0.0"/> <!--"-0.25"-->
        
        
    </node>
    
    
</launch>


